src/user/UserList.js

```diff
@@ -1,18 +1,50 @@
 import React from 'react';
 import { Link } from 'react-router';
+import ReduxInfiniteScroll from 'redux-infinite-scroll';
+import _ from 'lodash';
```

Comment:
Avoid using root better to use `import { isNumber, filter, take } from 'lodash';`